src/overlays/actors/ovl_En_Death/z_en_death.c

```diff
@@ -50,230 +69,1690 @@ ActorInit En_Death_InitVars = {
+    s8 index;
+
+    if (limbIndex == GOMESS_LIMB_SCYTHE_BLADE) {
+        Matrix_MultZero(&this->scytheWorldPos);
+
+        if ((this->weaponCollider.base.atFlags & AT_ON) && this->weaponColliderLastUpdateTime != this->actionTimer) {
+            Vec3f quad0;
+            Vec3f quad1;
+            Vec3f pos1;
+            Vec3f pos2;
+
+            Math_Vec3f_Copy(&quad0, &this->weaponCollider.dim.quad[0]);
+            Math_Vec3f_Copy(&quad1, &this->weaponCollider.dim.quad[1]);
+
+            if (this->actionFunc == EnDeath_EndSwingAttack) {
+                Matrix_MultVecX(6000.0f, &pos1);
+                Matrix_MultVec3f(&sWeaponColliderOffsetEndSwing, &pos2);
+            } else {
+                Matrix_MultVecY(5000.0f, &pos2);
+                Matrix_MultVec3f(&sWeaponColliderOffsetOther, &pos1);
+            }
+
+            Collider_SetQuadVertices(&this->weaponCollider, &pos2, &pos1, &quad0, &quad1);
+            EffectBlure_AddVertex(Effect_GetByIndex(this->effectIndex), &pos1, &pos2);
+
+            this->weaponColliderLastUpdateTime = this->actionTimer;
+        } else if (this->unk_18C) {
+            if (this->actionFunc == EnDeath_EndSwingAttack) {
+                Matrix_MultVecX(6000.0f, &this->weaponCollider.dim.quad[1]);
+                Matrix_MultVec3f(&sWeaponColliderOffsetEndSwing, &this->weaponCollider.dim.quad[0]);
+            } else {
+                Matrix_MultVecY(5000.0f, &this->weaponCollider.dim.quad[0]);
+                Matrix_MultVec3f(&sWeaponColliderOffsetOther, &this->weaponCollider.dim.quad[1]);
+            }
+            this->unk_18C = false;
+            this->weaponCollider.base.atFlags |= AT_ON;
+            this->weaponColliderLastUpdateTime = this->actionTimer;
+        }
+    } else if (limbIndex == GOMESS_LIMB_CORE_POS && this->actionFunc != EnDeath_Dead) {
+        Matrix_Push();
+        EnDeath_DrawCore(this, play);
+        Matrix_Pop();
+        Matrix_MultZero(&this->actor.focus.pos);
+
+        this->coreCollider.dim.worldSphere.center.x = this->actor.focus.pos.x;
+        this->coreCollider.dim.worldSphere.center.y = this->actor.focus.pos.y;
+        this->coreCollider.dim.worldSphere.center.z = this->actor.focus.pos.z;
+
+        if (this->actor.params < 5) {
+            this->coreCollider.dim.worldSphere.center.y = this->actor.focus.pos.y + 5.0f;
+        }
+    } else if (limbIndex == GOMESS_LIMB_SCYTHE_HANDLE) {
+        if (!(this->actionFunc != EnDeath_SpinAttack && this->actionFunc != EnDeath_BlockProjectile &&
+              this->actionFunc != EnDeath_IntroCutscenePart5) ||
+            (this->actionFunc == EnDeath_DeathCutscenePart1 && this->holdsScythe == true)) {
+            Matrix_Get(&this->scytheMtxF);
+        }
+    }
+
+    index = sDamageEffectPosMap[limbIndex];
+    if (index != -1) {
+        if (index < 7) {
+            // get matrix translation as-is, occupies slots 0, 1, 2, 3, 4, 5, 6
+            Matrix_MultZero(&this->dmgEffectPositions[index]);
```

Comment:
```suggestion
            Matrix_MultZero(&this->bodyPartsPos[index]);
```
Pretty sure we've named similar arrays like this, though this one is interesting in that a lot more slots are used for just effects instead of the usual 1 to 3 extra. Darkeye has been putting comments explaining this (see https://github.com/zeldaret/mm/blob/2a34685449dbb61bdfbef90d500703818809cc1a/src/overlays/actors/ovl_En_Bb/z_en_bb.c#L641). I'd probably still like to see it call it bodyPartsPos to keep the same terminology.